### ISSUE TITLE
Stops after step 1 for classification

### ISSUE THREAD

**Issue by jokc28** [dragonlzm/ISAL#7]
I've tried reproducing your work on classification task. 
Running the code went great, but it stopped working after finishing step 1. 
It doesn't seem to be my GPU's problem since I'm using 24G NVIDIA RTX A5000. 
There aren't any error messages showing up, not clearly telling me what the problem is. 
However, I did get a UserWarning message while running the code shown as below.
I really want to successfully re-implement your work and hope you can give me some insight in how to deal with the situation.
 
UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  pseudo_label = torch.nn.functional.softmax(preds).argmax().unsqueeze(dim=0)

P.S. Would it be possible to print influence value of the selected unlabeled sample layer-wise? It would be interesting if I could see how the influence value changes after each layer, but it doesn't seem like it is coded in such a way.

**Comment by dragonlzm** [comment]
Hi! For the first question, this warning may not be the reason the code stops running.
For your second question: yes, you could. The only thing that you need to do is to calculate the gradients of that specific layer, you can try to use the require_grad = False to freeze the rest of the layer. I think it could be an interesting thing to explore!

**Comment by dragonlzm** [comment]
I may close this issue now. If you have further questions, please don't hesitate to reopen it.
